<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Server</class>
 <widget class="QDialog" name="Server">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Server</string>
  </property>
  <widget class="QPushButton" name="StartB">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>10</y>
     <width>80</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Старт</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="StatusLI">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>10</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="PortHorizontalLayout">
    <item>
     <widget class="QLabel" name="PortL">
      <property name="text">
       <string>Порт</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="PortLI"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="ImageL">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>650</width>
     <height>480</height>
    </rect>
   </property>
   <property name="text">
    <string>                                                                                  Место появления изображения</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
